contents/blog/posthog-cloud-eu.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2022-10-10
+title: Introducing PostHog Cloud EU
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Product updates"]
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+---
+
+It's no secret that some popular analytics tools, such as Google Analytics, have fallen foul of GDPR regulations in the EU. That's one reason why we've enabled users to self-host PostHog, or to anonymize user data in a way which complies with GDPR. 
+
+Today, we're making a third option available: hosting PostHog Cloud in Europe so that user data never has to leave the EU. Now, any business which needs to comply with GDPR regulations can get up and running in a few minutes, and without the ongoing upkeep required for a self-hosted deployment. 
+
+Oh, and PostHog Cloud EU also happens to be faster for users located in Europe, which is nice. 
+
+PostHog Cloud EU offers everything you'd expect of PostHog - analytics, session recording, experiments and more. The only difference is that you can now choose to host in a different region, at no extra cost. 
+
+Until now, running PostHog with GDPR compliance meant self-hosting on your own infrastructure, or comprehensively anonymizing data if using PostHog Cloud.
+
+Thanks to our new hosting option, customers in the EU, and any business that needs to keep user data within the EU, can get up and running on PostHog Cloud EU in just a few minutes.
+
+<GDPRForm />
+
+## FAQ
+
+### Where are PostHog Cloud EU's servers based?
+
+PostHog Cloud EU is hosted in the AWS `eu-central-1` region based in Frankfurt, Germany.
+
+### What data is transferred from the EU to the US?
+
+None. PostHog Cloud EU is an entirely independent instance of PostHog. All event data, user data, and the product itself, is hosted on our EU-based infrastructure.
+
+### How do I use PostHog Cloud EU in compliance with GDPR?
+
+Just deploying PostHog Cloud EU isn't enough – you still need to [acquire consent](/tutorials/react-cookie-banner) from your users and comply with all other provisions of the GDPR, such as the right to be forgotten. Please see our [GDPR guidance](/docs/privacy/gdpr-compliance) for more information.
+
+### I'm an existing PostHog customer, can I migrate to PostHog Cloud EU?
+
+Yes. Please read our guide to [migrating to PostHog Cloud EU](/tutorials/migrate-eu-cloud). Please note: at this time, only event and person data can be migrated. Insights, dashboards, and feature flags are not migratable.
```

Comment:
```suggestion
[It's possible](/tutorials/migrate-eu-cloud), but **we do not recommend** migrating large, historical event volumes from existing PostHog instances as our Migrator app is primarily intended for use on smaller, hobby deployments. If you're interested in moving to PostHog Cloud EU from an existing PostHog deployment, please [contact our customer success team](URL HERE) for support.    
```

FAO @joethreepwood, @simfish85, @camerondeleone: Thoughts on the above? Where shall we send people interested in migrating? Is it worth us doing a form for this?

Reply:
I don't think we should do a form. That just prolongs the pain. We should send users who are interested to the tutorial, but do the best to give them info up front. 

I'd suggest using the warning on the tutorial here. E.g.

```suggestion
It is possible to migrate event data from another PostHog instance to EU Cloud, but it is a time-consuming process likely to take several days or weeks. There is currently no way to migrate other information, including insights, dashboards, user accounts, feature flags or API keys. 

We strongly recommend that users with large volumes of data begin by setting up and connect a new EU cloud instance and only attempt to migrate data later, if needed, by following [this migration tutorial](/tutorials/eu-cloud). 
```

Reply:
Yeah, to the tutorial?

Everywhere else on this page, we should drive to the landing page.